### PR TITLE
Broadcom AFBR update API to 1.5.6

### PR DESCRIPTION
**Pull request by AlexKlimaj**
This PR upgrades the Barodcom AFBR API to 1.5.6. It looks like all of the changes from 1.4.4 were under the hood.

https://broadcom.github.io/AFBR-S50-API/migration_guide.html

Multicopter
https://review.px4.io/plot_app?log=fff4bf3d-1eae-4eb3-92e8-09afb7b547a6

VTOL
https://review.px4.io/plot_app?log=062cf510-f140-4a0c-88c0-29e2e8d5f2ed
https://review.px4.io/plot_app?log=b63a3d1a-9c85-4001-bfaa-6a06ba2d45af

```
AFBR-S50 Chip ID: 33906, API Version: 17104902 v1.5.6
AFBR-S50LV85D

nsh> afbrs50 status
afbrs50: sample interval: 15978 events, 20166.85us avg, min 17349us max 25548us 1847.278us rms
distance: 1.874m
dfm mode: 2
rate: 50 Hz
nsh> listener distance_sensor

TOPIC: distance_sensor
 distance_sensor
    timestamp: 327245046 (0.027603 seconds ago)
    device_id: 18 (Type: 0x00, SPI:2 (0x00))
    min_distance: 0.08000
    max_distance: 30.00000
    current_distance: 1.87300
    variance: 0.00000
    h_fov: 0.10472
    v_fov: 0.10472
    q: [0.00000, 0.00000, 0.00000, 0.00000] (Roll: 0.0 deg, Pitch: -0.0 deg, Yaw: 0.0 deg)
    signal_quality: 100
    type: 0
    orientation: 25

nsh>

```

### CODE REVIEW

**Comment by dagar** [comment]
> Basic functionality is working but needs flight time.

Wait before merging?

**Comment by AlexKlimaj** [comment]
Lets hold off until I can get some flight time on it.

**Comment by AlexKlimaj** [comment]
I think this is ready to merge.

**Review by dagar — APPROVED** [review]

**Pull request merged**
